README_human.md

```diff
@@ -35,9 +63,26 @@ To download a version of the *human genome* with HiSat2 indexes, enter the follo
 ```
 
 ### Unpacking the Genome File
-The human genome file above is a compressed file and therefore needs to be unpacked.  *This step will be different if you have downloaded a different version of the human genome*
+The human genome file given above is a compressed file and therefore needs to be unpacked.  *This step will be different if you have downloaded a different version of the human genome*.
 To unpack the version given above, enter the command as follows:
 
 ```bash
 	tar -zxvf grch38.tar.gz
 ```
+
+## Load the HISAT2 Program
+This is done using the HISAT2 program, which can be loaded onto the cluster using the command:
```

Comment:
I think would be better to day "This is done using the module command which can be used to load programs for use on the cluster. To load the HISAT2 program use the following command"